src/tests/class_tests/openms/source/CrossLinksDB_test.cpp

```diff
@@ -47,6 +47,10 @@ struct ResidueModificationOriginCmp
 {
   bool operator() (const ResidueModification* a, const ResidueModification* b)
```

Comment:
I don't know if this operator is correct. Shouldn't it first test if pointers are equal and then if this is the same modification?